### PR TITLE
Remove unnecessary null coalescing operator

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs [hunk 1]
```diff
@@ -261,7 +261,7 @@ private async Task<TDocument> GetInternalAsync(bool failover = false)
                         return document;
                     }
 
-                    if (_isDistributed && (_options?.SlidingExpiration.HasValue ?? false))
+                    if (_isDistributed && _options.SlidingExpiration.HasValue)
```
Comment (Piedone) [inline src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs:264]: If `_options` is `null`, `_options?.SlidingExpiration.HasValue` will also be `null`.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs:264]: It wouldn't be null, check line 255

Comment (Piedone) [reply src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs:264]: I see. What about `SlidingExpiration` though?

Comment (hishamco) [reply src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs:264]: While we checking using `HasValue` nothing to worry, the old `??` operator was for `_option`

**Review by Piedone — APPROVED** [review]

**Pull request merged**